WatchApp Extension/Views/Carb Entry & Bolus/BolusInput.swift

```diff
@@ -42,8 +42,8 @@ struct BolusInput: View {
             DoseVolumeInput(
                 volume: amount,
                 isEditable: isEditable,
-                increment: { self.amount = self.pickerValues.incrementing(self.amount, by: 10) },
-                decrement: { self.amount = self.pickerValues.decrementing(self.amount, by: 10) },
+                increment: { self.amount = self.pickerValues[self.pickerValues.index(of: self.amount+0.5)] },
+                decrement: { self.amount = self.pickerValues[self.pickerValues.index(of: self.amount-0.5)] },
```

Comment:
With the combination of the crown being able to index over all values, and some pumps reporting greater gaps in their allowed bolus volumes list as bolus volumes get larger, I think this can end up selecting values that do not exist in the allowed volumes list. So index(of..) would return nil, and cause an exception.

Reply:
`index(of...)` will return the index before the `supportedVolumes.endIndex`(if there is more than 1 element in the `supportedVolumes` array) in the case that `supportedVolumes.firstIndex(where: { $0 > bolusValue })` is `nil`.

Reply:
Didn't realize it wasn't using the standard array index(of...). Will approve. Thanks.